### DISCUSSION TITLE
Tuning untuk Forecasting (Time Series)

### DISCUSSION THREAD

**Discussion by ChristoNindyo** [teamalgoritma/community#298] [Q&A] [answered]
Permisi, mas/mbak

Saya ingin bertanya mengenai tuning model seperti terlampir . Model saya sudah bisa menangkap pola (naik/turun), tapi secara nilai masih belum tepat. Apa yang bisa saya lakukan untuk meningkatkan performance ? Jika dengan tuning, variable apa yang mesti saya ubah. 

Jika misalkan ini sudah maksimal, apakah model hanya bisa dipakai untuk memprediksi trend saja ? 

![image](https://user-images.githubusercontent.com/105972495/189314578-c04dc154-2db1-46ba-95b4-f50af059cd00.png)

**Comment by NabiilahArdini** [comment, marked as answer]
Dear @ChristoNindyo ,

Dari plot yang Bapak miliki, model sepertinya sudah menangkap kurang lebih efek trend, seasonal, dan error dari data. Namun ketika memprediksi, trend yang dibuat masih lebih rendah dibandingkan data aktualnya. Hal ini kemungkinan karena ia kurang mengikuti pola trend yang terbaru (masih cukup mempertimbangkan data lampau).

Untuk meningkatkannya, bila Bapak menggunakan model exponential smoothing, Bapak bisa tuning nilai alpha (untuk trend) agar lebih tinggi. Bila nilainya lebih tinggi maka dia semakin mempertimbangkan trend terbaru dan kurang mempertimbangkan trend yang lampau.

Semoga dapat membantu. Apabila jawaban ini membantu Bapak, bisa berikan "Mark as answer" pada comment ini.

**Reply by ChristoNindyo** [reply]
Terima kasih atas jawabannya, mbak
